### PR TITLE
new branch

### PR DESCRIPTION
**Pull request by LokeshSrinivasa**

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+sbt.version=0.13.17
```
Comment (glidester) [inline project/build.properties:1]: Need empty line at end of file

File: src/test/resources/test-data/request.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "data": {
+    "id": 2,
+    "email": "janet.weaver@reqres.in",
+    "first_name": "Janet",
+    "last_name": "Weaver",
+    "avatar": "https://s3.amazonaws.com/uifaces/faces/twitter/josephstein/128.jpg"
+  }
+}
```
Comment (glidester) [inline src/test/resources/test-data/request.json:9]: Need empty line

File: src/test/scala/uk/gov/hmrc/runner/RunQA.scala [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+@RunWith(classOf[Cucumber])
+@CucumberOptions(
+  features = Array("src/test/resources/features"),
+  glue = Array("uk.gov.hmrc.stepdefs"),
+  plugin = Array("pretty", "html:target/cucumber", "json:target/cucumber.json",
+    "uk.gov.hmrc.extentreport.ExtentCucumberFormatter:target/test-reports/html-report/index.html"),
+  tags=Array("@qa")
+)
+class RunQA { }
```
Comment (glidester) [inline src/test/scala/uk/gov/hmrc/runner/RunQA.scala:16]: Need empty line

File: src/test/scala/uk/gov/hmrc/runner/RunWip.scala [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+import org.junit.runner.RunWith
+
+@RunWith(classOf[Cucumber])
+@CucumberOptions(
+features = Array("src/test/resources/features"),
+glue = Array("uk.gov.hmrc.stepdefs"),
+plugin = Array("pretty", "html:target/cucumber", "json:target/cucumber.json"),
+tags=Array("@wip")
+)
+class RunWip { }
```
Comment (glidester) [inline src/test/scala/uk/gov/hmrc/runner/RunWip.scala:14]: Need empty line

**Review by glidester — CHANGES_REQUESTED** [review]

File: src/test/scala/uk/gov/hmrc/runner/RunWip.scala (hunk 4 above)
Comment (LokeshSrinivasa) [reply src/test/scala/uk/gov/hmrc/runner/RunWip.scala]: added an empty line at the end of file.

File: project/build.properties (hunk 1 above)
Comment (LokeshSrinivasa) [reply project/build.properties]: added an empty line at the end of file.

File: src/test/resources/test-data/request.json (hunk 2 above)
Comment (LokeshSrinivasa) [reply src/test/resources/test-data/request.json]: added an empty line at the end of file.

File: src/test/scala/uk/gov/hmrc/runner/RunQA.scala (hunk 3 above)
Comment (LokeshSrinivasa) [reply src/test/scala/uk/gov/hmrc/runner/RunQA.scala]: added an empty line at the end of file.

**Review by glidester — APPROVED** [review]

**Pull request merged**